src/cmp/cmp.go

```diff
@@ -31,9 +31,11 @@ func Less[T Ordered](x, y T) bool {
 
 // Compare returns
 //
-//	-1 if x is less than y,
-//	 0 if x equals y,
-//	+1 if x is greater than y.
+// -1 if x is less than y,
```

Comment:
this is not correct - edge case of `go fmt`. best fix would probably be to move the next line left one space relative to its original position